CHANGELOG

```diff
@@ -7,11 +7,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [1.3.9] 2025-01-07
+
 ### Updated
 
 - Updated phone, email, notification preference and home library to be individually editable in Account Settings (SCC-4337, SCC-4254, SCC-4253)
 - Updated username to be editable in My Account header (SCC-4236)
 
+## [1.3.8] 2024-12-9
+
+Note this release was not merged to main before deploying.
```

Comment:
Can you delete this? NBD that it's in prod but this won't be true once you completely backmerge.